src/OpenLoco/src/S5/S5.cpp

```diff
@@ -226,26 +238,31 @@ namespace OpenLoco::S5
         return file;
     }
 
-    static constexpr bool shouldPackObjects(uint32_t flags)
+    static constexpr bool shouldPackObjects(SaveFlags flags)
     {
-        return !(flags & SaveFlags::raw) && !(flags & SaveFlags::dump) && (flags & SaveFlags::packCustomObjects) && !isNetworked();
+        return (flags & SaveFlags::raw) == SaveFlags::none
+            && (flags & SaveFlags::dump) == SaveFlags::none
+            && (flags & SaveFlags::packCustomObjects) != SaveFlags::none
```

Comment:
Should we use `hasSaveFlags` for these as well?

Reply:
Ah, those functions are methods, so unavailable without making changes to what's passed in as a parameter. Let's leave it then.